hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/OzoneQuota.java

```diff
@@ -43,13 +43,25 @@
   public static final String OZONE_QUOTA_TB = "TB";
 
   /** Quota Units.*/
-  public enum Units {UNDEFINED, BYTES, KB, MB, GB, TB}
+  public enum Units {BYTES, KB, MB, GB, TB}
 
   // Quota to decide how many buckets can be created.
   private long quotaInCounts;
   // Quota to decide how many storage space will be used in bytes.
   private long quotaInBytes;
   private RawQuotaInBytes rawQuotaInBytes;
+  // Data class of Quota.
+  private static QuotaList quotaList;
```

Comment:
we can Initialize quotaList in static {} 
```suggestion
  private static QuotaList quotaList;
  static {
     quotaList = new QuotaList();
     quotaList.addQuotaList(OZONE_QUOTA_TB, Units.TB, TB);
     quotaList.addQuotaList(OZONE_QUOTA_GB, Units.GB, GB);
     quotaList.addQuotaList(OZONE_QUOTA_MB, Units.MB, MB);
     quotaList.addQuotaList(OZONE_QUOTA_KB, Units.KB, KB);
     quotaList.addQuotaList(OZONE_QUOTA_BYTES, Units.BYTES, 1L);
  }
```